nntrainer/tensor/blas_neon.h

```diff
@@ -15,6 +15,7 @@
 #ifdef __cplusplus
 
 #include <arm_neon.h>
+#include <vector>
```

Comment:
```suggestion
```

remove?

Reply:
I intentionally leaved it since it is used in blas_neon.cpp
but I also think that it is better to be moved to .cpp file